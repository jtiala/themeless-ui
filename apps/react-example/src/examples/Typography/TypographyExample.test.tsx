import {
  componentToJson,
  describe,
  expect,
  it,
  render,
  renderer,
  screen,
} from "../../../test";
import { TypographyExample } from "./TypographyExample";

describe("TypographyExample", async () => {
  it("should render the example", () => {
    expect(
      componentToJson(renderer.create(<TypographyExample />)),
    ).toMatchSnapshot();

    render(<TypographyExample />);

    const component = screen.getByTestId("typographyExample");

    expect(component).toBeInTheDocument();
  });
});
