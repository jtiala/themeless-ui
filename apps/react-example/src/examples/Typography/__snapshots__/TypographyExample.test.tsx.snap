// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`TypographyExample > should match the snapshot 1`] = `
<article
  data-testid="typographyExample"
>
  <h1
    className="tui__heading"
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit
  </h1>
  <p
    className="tui__paragraph"
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam lacinia cursus ex, in lacinia est.
     
    <strong
      className="tui__text"
    >
      Vivamus rhoncus mollis
    </strong>
     felis nec efficitur. Maecenas id malesuada quam. Suspendisse aliquam urna sed posuere malesuada. Ut viverra ipsum eget ipsum eleifend dignissim. Nullam vitae nunc quam. Ut aliquam pharetra nunc, eget posuere nisi viverra sed. Vivamus nec nibh at est ultrices laoreet volutpat ut sapien. Sed scelerisque augue nec ex interdum, in tincidunt nunc imperdiet. Sed lacinia vulputate sem eu pharetra. Ut faucibus dui urna, ac maximus eros finibus eget. Cras imperdiet odio non erat egestas sagittis. Donec eget odio turpis. Curabitur dictum libero in velit ultrices sollicitudin. Mauris semper vulputate consectetur.
  </p>
  <p
    className="tui__paragraph"
  >
    Suspendisse accumsan rhoncus neque. Maecenas molestie sollicitudin semper. Nullam pulvinar sollicitudin justo. Sed sollicitudin lorem at est rutrum, a suscipit lorem venenatis.
     
    <mark
      className="tui__text"
    >
      In volutpat, ex eget tempus auctor, lorem turpis finibus dui, et ultrices urna orci eget lorem. Suspendisse mauris tellus, mattis id feugiat vel, imperdiet quis massa. Integer non lacus est.
    </mark>
     
    Suspendisse nunc augue, tristique at enim vel, sollicitudin maximus ante. Quisque rutrum tempor velit non varius. Suspendisse ornare mauris at mi vulputate fringilla.
  </p>
  <h2
    className="tui__heading"
  >
    Cras ut ipsum ut erat semper gravida
  </h2>
  <p
    className="tui__paragraph"
  >
    Cras ut ipsum ut erat semper gravida. Nam suscipit varius tincidunt. Vestibulum vulputate sed dolor nec laoreet. Cras enim nulla, bibendum eu placerat non, consequat at erat. Nam eleifend maximus faucibus. Praesent sagittis sem non urna consectetur, non fringilla mi posuere. Sed non mi tortor. 
    <em
      className="tui__text"
    >
      Cras efficitur blandit rhoncus.
    </em>
     Cras ullamcorper non diam nec sagittis. Nunc eget lorem eu ex rutrum molestie sed vitae augue. Maecenas in risus libero. Fusce sapien lacus, tristique eget justo sed, auctor fringilla metus. Mauris augue nunc, lobortis in lectus vel, faucibus posuere velit.
  </p>
  <blockquote
    cite="https://source-of-the-quote.example.com"
    className="tui__blockquote"
  >
    <div
      className="tui__blockquote-quote"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec accumsan ex vel lacinia volutpat. Ut ultricies, mauris et varius finibus, nulla ante efficitur libero, eget convallis odio ex ut neque. Praesent quis odio ac felis pretium pharetra sit amet quis neque. Nam nec libero purus. Maecenas sagittis lobortis lacinia. Aliquam non vehicula metus. Vivamus fringilla ante eget leo pretium cursus. Morbi venenatis diam metus, ut faucibus dolor mollis a. Nulla nec ex sapien. Curabitur a tempor est. Mauris faucibus congue mi, sit amet pulvinar justo porttitor nec. Nunc et eros a est fermentum lacinia id at nisi.
    </div>
    <footer>
      <span
        className="tui__blockquote-author"
      >
        â€”
        Samuel L. Ipsum
      </span>
      <span>
        , 
      </span>
      <cite
        className="tui__blockquote-source"
      >
        Lorem Ipsum
      </cite>
    </footer>
  </blockquote>
  <p
    className="tui__paragraph"
  >
    Etiam porta purus a aliquam rutrum. Vivamus finibus sed metus et vestibulum. Ut ornare eget mi nec congue. Nunc et dui sed diam viverra porta. Phasellus gravida massa lorem, et ultrices tellus scelerisque eget. Cras congue congue pulvinar. Aliquam aliquet dolor est, eget lobortis tortor efficitur ut. Nam accumsan, nisl quis pretium imperdiet, leo nulla efficitur lectus, eget vestibulum diam lorem id ante. Cras sit amet cursus purus, eget bibendum arcu.
  </p>
  <h2
    className="tui__heading"
  >
    Cras sit amet cursus purus
  </h2>
  <h3
    className="tui__heading"
  >
    Sed sodales eros
  </h3>
  <p
    className="tui__paragraph"
  >
    Sed sodales eros ut arcu tincidunt, et condimentum lacus ultricies. Quisque rhoncus, nulla et vulputate vestibulum, ligula dolor tristique massa, sollicitudin fermentum eros magna nec est. Nulla facilisi. Quisque efficitur turpis sit amet ornare euismod. Pellentesque in neque vitae risus posuere condimentum at in ex. Morbi consequat erat at risus laoreet rhoncus. Suspendisse pharetra bibendum ante eu gravida. Pellentesque interdum eget tortor sed venenatis. Suspendisse mollis turpis non justo sollicitudin convallis. Ut quis eros lacus. Cras aliquam non ex ultrices sollicitudin. Suspendisse efficitur id lorem ut gravida. Morbi laoreet nec magna molestie eleifend. Proin ligula risus, tristique et nunc et, sagittis aliquet lacus. Duis sollicitudin elit vitae nisl rhoncus pretium. Donec at auctor neque.
  </p>
  <p
    className="tui__paragraph"
  >
    Nunc imperdiet, dui et molestie congue, odio velit imperdiet lacus, at aliquet nulla urna sit amet orci. Praesent sed dolor luctus, ornare metus id, iaculis ante. Integer non suscipit nunc, eu cursus risus. Nulla vel iaculis est, et viverra metus. Mauris venenatis sodales lacus ultricies scelerisque. Praesent lobortis felis purus, non vulputate leo sollicitudin vitae. In a felis non nunc congue volutpat ut sed orci. Duis volutpat tristique neque ac sodales. Duis faucibus maximus lectus, a aliquam nisi pharetra nec. Sed risus tortor, sollicitudin sed malesuada a, imperdiet vitae nisi. Curabitur ut hendrerit ex. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Nam tellus nisi, tincidunt vitae justo ac, scelerisque feugiat dolor. Proin vitae erat pretium, luctus lorem eu, imperdiet augue.
  </p>
  <h4
    className="tui__heading"
  >
    Vivamus lacinia semper dolor
  </h4>
  <p
    className="tui__paragraph"
  >
    Vivamus lacinia semper dolor, et vehicula velit condimentum sed. Proin consequat tellus ac arcu tincidunt aliquet. Ut bibendum sodales justo, in malesuada nulla semper non. Morbi lectus diam, tincidunt varius magna id, congue semper ipsum. Vestibulum nisi eros, aliquet mollis scelerisque ac, aliquam non magna. Suspendisse nec mi cursus, fermentum velit nec, sollicitudin enim. Maecenas elementum in velit vitae viverra. Etiam ut consequat lorem, eget posuere dui. Sed enim massa, ornare ac ipsum venenatis, convallis luctus arcu. Ut vel consectetur arcu.
  </p>
  <p
    className="tui__paragraph"
  >
    Suspendisse convallis mi sed metus consectetur euismod. Proin vel libero nisi. Curabitur nec nunc eget mauris tempus consectetur. Sed et elit at enim luctus imperdiet sollicitudin eu diam. Proin vitae nisi ornare, ullamcorper felis sit amet, placerat leo. Integer sapien mauris, cursus fringilla erat quis, commodo pharetra nulla. Vivamus sit amet hendrerit dui, eu egestas dolor. Donec interdum convallis velit ut pellentesque. Quisque eu ultrices nisi. Integer eleifend ante ut scelerisque porta. Aenean accumsan eget velit vestibulum iaculis. Praesent elementum fermentum massa vitae sollicitudin. Sed eget maximus leo, ac aliquam quam.
  </p>
  <h5
    className="tui__heading"
  >
    Integer maximus sapien
  </h5>
  <p
    className="tui__paragraph"
  >
    Integer maximus sapien a libero venenatis, id vulputate leo molestie. Suspendisse pellentesque, turpis vel vehicula varius, quam purus dapibus orci, at molestie tortor massa ut purus. Maecenas sit amet vulputate ex. Suspendisse nec nibh vitae erat rhoncus semper feugiat et diam. Suspendisse potenti. Cras neque tortor, consequat eu enim in, suscipit dapibus quam. Quisque eget pulvinar dolor. Etiam porta tincidunt velit, in laoreet elit tincidunt a. Nulla volutpat eleifend dictum.
  </p>
  <p
    className="tui__paragraph"
  >
    Nulla scelerisque nulla leo, et lacinia est feugiat eu. Morbi dignissim nulla lectus, in fermentum felis ullamcorper nec. Proin at ligula eros. Ut non erat gravida orci tempus fringilla vitae nec nisl. In varius, dui scelerisque blandit malesuada, erat ante placerat ipsum, eget malesuada neque leo vitae tortor. Praesent eleifend felis velit, id egestas lorem blandit id. Nulla pulvinar nisi malesuada mauris tempus, ac lobortis dolor sagittis.
  </p>
  <h6
    className="tui__heading"
  >
    Cras fermentum porta
  </h6>
  <p
    className="tui__paragraph"
  >
    Cras fermentum porta enim, at pellentesque diam egestas sit amet. Sed sed auctor odio. Donec turpis purus, aliquam non nisi quis, mollis tempor metus. Phasellus cursus nulla et ipsum vehicula, fringilla aliquam nisi venenatis. Donec consequat venenatis nisl at egestas. Proin scelerisque sagittis turpis. Curabitur gravida, purus non congue condimentum, urna mauris fringilla ante, sed finibus ante tellus vitae lectus. Pellentesque in imperdiet ligula.
  </p>
  <p
    className="tui__paragraph"
  >
    Etiam rutrum mi sit amet pulvinar suscipit. Praesent quis enim ac orci elementum tempor congue sodales erat. Suspendisse potenti. Proin et feugiat quam. Mauris vel laoreet lectus, sit amet bibendum dolor. Fusce ut odio semper, tincidunt tortor et, imperdiet velit. Donec lacinia diam a enim porttitor, sit amet blandit nisl congue. Praesent cursus lorem eget vehicula auctor. Praesent tincidunt interdum elementum. Vestibulum vel accumsan lacus. Integer eget volutpat nibh, nec ornare nulla. Duis id velit pulvinar, finibus nunc eget, ullamcorper libero. Vivamus leo ex, interdum in auctor non, tempor vehicula dui. Vestibulum sed efficitur eros. Donec lacinia ultrices eros vel mattis.
  </p>
</article>
`;
